mpqp/tools/display.py

```diff
@@ -143,7 +143,10 @@ def format_element(element: Union[int, float, complex | Expr], round: int = 5) -
 
 @typechecked
 def clean_1D_array(
-    array: list[complex] | npt.NDArray[np.complex64 | np.float32], round: int = 5
+    array: (
+        list[complex | float | int] | npt.NDArray[np.complex64 | np.float32 | np.int32]
```

Comment:
Maybe instead of putting `complex | float | int`, we can use the class `Complex` from the package `numbers`?